readme/apps/debugging.md

```diff
@@ -2,6 +2,16 @@
 
 It is possible to get the apps to display or log more information that might help debug various issues.
 
+:::note
+
+Before posting a full log to GitHub or the forum, be aware that it may contain (but isn't limited to):
+- Note/notebook/tag titles
```

Comment:
It **might** make sense to also update the GitHub issue template (and the forum template) with this information.